templates/default/openssh.conf.erb

```diff
@@ -117,3 +117,10 @@ UseRoaming <%= @node['ssh-hardening']['ssh']['client']['roaming'] ? 'yes' : 'no'
 # Send locale environment variables
 SendEnv <%= @node['ssh-hardening']['ssh']['client']['send_env'].join(' ') %>
 <% end %>
+
+<%- unless Array(@node['ssh-hardening']['ssh']['client']['extras']).empty? %>
```

Comment:
@bdwyertech you don't have to use `Array()`, the value is defined as Hash in the default attributes. You can do ` unless @node[...].empty?`